modules/auxiliary/gather/apache_superset_priv_esc.rb

```diff
@@ -0,0 +1,181 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HTTP::FlaskUnsign
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Apache Superset Signed Cookie Priv Esc',
+        'Description' => %q{
+          Apache Superset versions <= 2.0.0 utilize Flask with a known default secret key which is used to sign HTTP cookies.
+          These cookies can therefore be forged. If a user is able to login to the site, they can decode the cookie, set their user_id to that
+          of an administrator, and re-sign the cookie. This valid cookie can then be used to login as the targeted user and retrieve database
+          credentials saved in Apache Superset.
+        },
+        'Author' => [
+          'h00die', # MSF module
+          'paradoxis', #  original flask-unsign tool
+          'zeroSteiner', # MSF flask-unsign library
```

Comment:
```suggestion
          'Spencer McIntyre', # MSF flask-unsign library
```

If you don't mind :smile: